apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-welcome-tour/src/tour-content.js

```diff
@@ -72,7 +72,7 @@ function getTourContent() {
 		},
 		{
 			heading: __( 'Drag & drop', 'full-site-editing' ),
-			description: __( 'To move blocks around click and drag the handle.', 'full-site-editing' ),
+			description: __( 'To move blocks around, click and drag the handle.', 'full-site-editing' ),
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.